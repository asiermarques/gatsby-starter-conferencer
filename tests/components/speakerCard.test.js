import React from "react"
import renderer from "react-test-renderer"

import SpeakerCard from "../../src/components/speakerCard"
import SpeakerFixture from "../__fixtures/Speaker"

describe("SpeakerCard", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SpeakerCard speaker={SpeakerFixture}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})