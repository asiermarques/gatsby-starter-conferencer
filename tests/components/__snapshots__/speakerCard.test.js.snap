// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpeakerCard renders correctly 1`] = `
<a
  class="card speaker"
  href="sara-smith"
>
  <div
    class="image"
    style={
      Object {
        "backgroundImage": "url('/assets/img/test.jpg')",
      }
    }
  />
  <div
    class="card-body"
  >
    <h5
      class="card-title"
    >
      Sara Smith
    </h5>
    <p
      class="card-text"
    >
      CTO Tyller Corp.
    </p>
  </div>
</a>
`;
